postpic/datahandling.py

```diff
@@ -627,6 +627,35 @@ def __copy__(self):
             ret.__dict__[k] = copy.copy(self.__dict__[k])
         return ret
 
+    def __deepcopy__(self, memo=None):
+        '''
+        returns a deep copy of the object.
+        This method is called by `copy.deepcopy(obj)`.
+
+        Although Field objects are mostly immutable, they have a __setitem__, which is useful
+        in some cases. Sometimes, one needs an easy way to copy a Field before using __setitem__.
+        This function copies the underlying data to make sure that no modifications applied to
+        the copy affect the source.
+
+        As the contents of `axes` should be strictly immutable, they don't need to be copied.
+        '''
+        ret = copy.copy(self)
+        ret.matrix = self.matrix.copy()
+        return ret
+
+    def copy(self, deep=True):
```

Comment:
Im prinzip benutzt sich das Field ja wie ein numpy array, nur mit Achsen. Da waere es doch nuetzlich wenn die Signatur der copy funktion, die Signatur der ndarray.copy funktion abbildet:
https://docs.scipy.org/doc/numpy/reference/generated/numpy.copy.html

Reply:
Ich habe auch kurz überlegt, ob es sinnvoll ist das Argument `order` hier zu haben. Aber dann könnte ich nicht einfach in `Field.copy` auf `copy.copy` und `copy.deepcopy` zurückgreifen, weil ich das da nicht durchreichen kann. Dann müsste ich die ganze Logik umdrehen: `Field.copy` für beide Varianten implementieren und das dann von `Field.__copy__` und `Field.__deepcopy__` aus aufrufen. Da war mir das `order` Argument mit echt marginalem Nutzen den Aufwand nicht wert.

Reply:
Das sehe ich ein. Dennoch sorgt es dafuer dass man ein `Field` nicht einfach wie ein`numpy.ndarray` behandeln kann. Ich mache ein issue auf, dass man das mal irgendwann in der zukunft fixen sollte.